packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -1026,64 +1026,78 @@ export function getSecurityRequirements(
   return results
 }
 
+export enum CaseStyle {
```

Comment:
These are the different cases that we support. I wonder if I should keep the enum values themselves consistent instead of trying to replicate the cases.

Reply:
It's a bit quirky trying to emulate the cases, but drives home the point what they are.